src/RWA007-A/scripts/RWA007Deployment.s.sol

```diff
@@ -0,0 +1,174 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.6.12;
+
+import "forge-std/Script.sol";
+import "../contract/Import.sol";
```

Comment:
You can probably inline the imports here instead of having them in a separate file.
The only reason I added this file in the past was to force `foundry` to pick the contracts and compile them.

Reply:
done